safe-global/web-core
# feat: connect list, propose and confirm endpoints

## What it solves

Resolves #1159

## How this PR fixes it

The `safe-react-gateway-sdk` has been updated and all of the types accordingly. The list, propose and confirm endpoints have been connected to the relevant hooks: `useLoadSafeMessages`, `useMessages` and the dispatchers accordingly.

## How to test it

Programmatically call `dispatchSafeMsgProposal` and observe the new message in the list. Confirmation can be done via `dispatchSafeMsgConfirmation`.

Safe `gor:0xF3E977e7Eea1A91ce5B2d5077e5A7195Ae18b722` already has a submitted message.

Note: the gateway is fixing issues surrounding EIP-712 messages so only EIP-191 messages are working at the moment.
